websockets/client.py

```diff
@@ -30,9 +31,17 @@ class WebSocketClientProtocol(WebSocketCommonProtocol):
 
     def __init__(self, *,
                  origin=None, extensions=None, subprotocols=None,
-                 extra_headers=None, **kwds):
+                 extra_headers=None, use_compression=True, **kwds):
         self.origin = origin
-        self.available_extensions = extensions
+        self.available_extensions = []
+        if use_compression:
+            self.available_extensions.append(
+                'permessage-deflate; '
+                'client_no_context_takeover; '
```

Comment:
I don't think this is correct, because our implementation retains the sliding window between messages.